src/ui/components/Button/index.js

```diff
@@ -9,31 +9,32 @@ import './Button.scss';
 
 export default class Button extends React.Component {
   static propTypes = {
-    appearance: PropTypes.string,
```

Comment:
Using boolean props or a single appearance prop are both fine with me. I would just like to not see hard coded class names.